src/site-recovery/azext_site_recovery/aaz/latest/site_recovery/fabric/_create.py

```diff
@@ -0,0 +1,1604 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "site-recovery fabric create",
+)
+class Create(AAZCommand):
+    """Create operation to create an Azure Site Recovery fabric (for e.g. Hyper-V site).
+
+    :example: fabric create for A2A
+        az site-recovery fabric create -n {fabric1_name} -g {rg} --vault-name {vault_name} --custom-details '{azure:{location:eastus}}'
```

Comment:
Please don't use "{}" as a placeholder in example. You should use the fake value instead. Because it will be confused with the shorthand syntax

Reply:
Agreed. Will remove.